src/main/resources/application.properties

```diff
@@ -61,3 +61,4 @@ oauth.redirectUris=http://localhost
 oauth.clientId=3bc8282c-7730-11e5-8bcf-feff819cdc9f
 oauth.clientSecret=6035bd78-7730-11e5-8bcf-feff819cdc9f
 unitDatabase.url=http://spooky.github.io/unitdb/#/
+login.create.accountUrl=https://www.faforever.com/account/register
```

Comment:
This property reads like:
In the section `login` there is a section `create` and in there is a property `accountUrl`, which is semantically not correct (It's not the "accountUrl").
I recommend `login.createAccountUrl`, `createAccount.url` or probably just `createAccountUrl`.

I will turn properties into YAML files in future, there it will be more obvious what makes sense:
```
login:
  create:
    accountUrl: ...
```
vs:
```
createAccountUrl: ...
```